packages/dev/core/src/Behaviors/Meshes/pointerDragBehavior.ts

```diff
@@ -503,14 +504,27 @@ export class PointerDragBehavior implements Behavior<AbstractMesh> {
             }
         }
 
-        const pickResult = PointerDragBehavior._PlaneScene.pickWithRay(ray, (m) => {
-            return m == this._dragPlane;
-        });
-        if (pickResult && pickResult.hit && pickResult.pickedMesh && pickResult.pickedPoint) {
-            return pickResult.pickedPoint;
-        } else {
+        // use an infinite plane instead of ray picking a mesh that must be updated every frame
+        const planeNormal = this._dragPlane.forward;
+        const planePosition = this._dragPlane.position;
+        const dotProduct = ray.direction.dot(planeNormal);

```

Comment:
Just to be sure: are we sure that `planeNormal` and `ray.direction` are normalized?

Reply:
Yes.